src/Views/AlertList/Parsing.elm

```diff
@@ -1,6 +1,6 @@
-module Alerts.Parsing exposing (..)
+module Views.AlertList.Parsing exposing (alertsParser)
```

Comment:
My initial thought when reading this is "`alertsParser` isn't the responsibility of the view".

I'm thinking it might make more sense for this to live somewhere in the top level app, and then it knows which view to render?